grl/grad.py

```diff
@@ -0,0 +1,45 @@
+import logging
+
+import numpy as np
+
+from jax import grad
+
+def do_grad(pe, pi_abs, no_gamma, lr=1):
```

Comment:
Can you change `pe` to `policy_eval` or something more verbose?

Reply:
Done